### PR TITLE
improve documentation format

### PR DESCRIPTION
**Pull request by hboutemy**
these improvements are useful in master branch, whatever happens to current discussions on testing and eventually changing some structure

### CODE REVIEW

**Comment by hboutemy** [comment]
@michael-o

**Comment by michael-o** [comment]
Will have a look next week.

**Comment by michael-o** [comment]
@ok2c HC Site is not forgotten, but lockdown is giving me a hard time right now. I will try to engage next week.

File: src/site/site.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```
Comment (ok2c) [inline src/site/site.xml:1]: @hboutemy It looks like Apache RAT chokes on this file
```
[INFO] --- apache-rat-plugin:0.13:check (default) @ httpclient5-parent ---
[INFO] Enabled default license matchers.
[INFO] Will parse SCM ignores for exclusions...
[INFO] Parsing exclusions from /home/travis/build/apache/httpcomponents-client/.gitignore
[INFO] Finished adding exclusions from SCM ignore files.
[INFO] 76 implicit excludes (use -debug for more details).
[INFO] 9 explicit excludes (use -debug for more details).
[INFO] 16 resources included (use -debug for more details)
[INFO] Rat check: Summary over all files. Unapproved: 1, unknown: 1, generated: 0, approved: 8 licenses.
[INFO] ------------------------------------------------------------------------
```

Comment (hboutemy) [reply src/site/site.xml:1]: license header updated: now rat is ok

**Comment by ok2c** [comment]
@hboutemy Unfortunately the build is still failing.

**Pull request merged**